### DISCUSSION TITLE
[Feature Request] Zombie Villager cure discounts apply to all players

### DISCUSSION THREAD

**Discussion by Bobbyperson** [PurpurMC/Purpur#741] [__Dismissed__] [answered]
Hello, I would like to request the option for cured zombie villagers to give discounts to not just one player, but everyone. The players on my server have been adamantly requesting this feature for some time now but I cannot find a plugin to fill this role. If it's not too much work, I would be truly thankful if you could aid me in this situation. I'm sure many other individuals would greatly appreciate this feature as well.

Thank you for your time, and I hope to hear back from you soon!

**Comment by VonSparq** [comment, marked as answer]
This was suggested before (#680) but I would love for the feature to be added.

**Comment by encode42** [comment]
As above, duplicate request
